scripts/visualize/visualize_map.R

```diff
@@ -58,11 +58,22 @@ visualize.states_svg <- function(viz){
     
     # why can't xml2 allow me to just move the node to be under the group?
     xml_add_child(xml_add_child(gb, 'g', transform=transform), 
-                  'use', 'xlink:href'=paste0("#", id.use), id=paste0(id.name,'-background'), class='state-background')
+                  'use', 'xlink:href'=paste0("#", id.use), id=paste0(id.name,'-background'), class='state-background', 
+                  onmousemove=sprintf("hovertext('%s',evt);", state.name[i]),
+                  onmouseout="hovertext(' ');")
     xml_add_child(xml_add_child(gf, 'g', transform=transform), 
-                  'use', 'xlink:href'=paste0("#", id.use), id=id.name, class='state-foreground')
+                  'use', 'xlink:href'=paste0("#", id.use), id=id.name, class='state-foreground',
+                  onmousemove=sprintf("hovertext('%s',evt);", state.name[i]),
```

Comment:
we are going to want to do something w/ a string replace since those state names are all lowercase. Should be fine for now.